modular_skyrat/modules/primitive_catgirls/code/map_items.dm

```diff
@@ -23,12 +8,92 @@
 	grill = TRUE
 	add_overlay("bonfire_grill")
 
-// Dirt but icebox
+// Dirt but icebox and also farmable
 
 /turf/open/misc/dirt/icemoon
 	baseturfs = /turf/open/openspace/icemoon
 	initial_gas_mix = "ICEMOON_ATMOS"
 
+/turf/open/misc/dirt/icemoon/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/simple_farm, set_plant = TRUE)
+
+// Water that can be fished out of
+
+/turf/open/water/icebox_fishable
+	desc = "Water kept warm through some unknown heat source, possibly a geothermal heat source far underground. \
+		Whatever it is, it feels pretty damn nice to swim in given the rest of the environment around here, and you \
+		can even catch a glimpse of the odd fish darting through the water."
+	baseturfs = /turf/open/openspace/icemoon
+	initial_gas_mix = "ICEMOON_ATMOS"
+	/// Holder for the steam particles that show up sometimes
+	var/obj/effect/abstract/particle_holder/particle_effect
+
+/turf/open/water/icebox_fishable/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/lazy_fishing_spot, /datum/fish_source/icecat_hot_spring)
+	if(prob(60))
+		particle_effect = new(src, /particles/hotspring_steam)
+
+/turf/open/water/icebox_fishable/Destroy()
+	QDEL_NULL(particle_effect)
+	return ..()
+
+/turf/open/water/icebox_fishable/Entered(atom/movable/arrived)
+	..()
+	wash_atom(arrived)
+	wash_atom(loc)
+
+// Cleans the given atom of whatever dirties it
+/turf/open/water/icebox_fishable/proc/wash_atom(atom/nasty)
+	nasty.wash(CLEAN_WASH)
+
+/turf/open/water/icebox_fishable/Entered(atom/movable/arrived)
+	..()
+	if(istype(arrived, /mob/living))
+		hotspring_mood(arrived)
+
+// Applies the hot water mood buff on the passed mob
+/turf/open/water/icebox_fishable/proc/hotspring_mood(mob/living/swimmer)
+	swimmer.add_mood_event("hotspring", /datum/mood_event/hotspring/nerfed)
+
+/datum/mood_event/hotspring/nerfed
+	description = span_nicegreen("The water was enjoyably warm!\n")
+	mood_change = 2
+
+// Steam particles for pairing with the hotsprings above
+
+/particles/hotspring_steam
+	icon = 'icons/effects/particles/smoke.dmi'
+	icon_state = list(
+		"steam_1"=2,
+		"steam_2"=2,
+		"Steam_3"=1,
```

Comment:
```suggestion
		"steam_1" = 2,
		"steam_2" = 2,
		"Steam_3" = 1,
```

Reply:
Also, the last one is capitalized, whilst the others aren't. Is that intended?

Reply:
it is not